### ISSUE TITLE
Auto Complete Environment Variable Name

### ISSUE THREAD

**Issue by SwapnilkumarRane** [usebruno/bruno#2161]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Most of the time we tend to reuse Environment Variables in our API request, Sometime it might happen we misspell the variable name and due to the same request gets failed, if we can have a feature that will display the list of available Environment variables defined, it will not only speed up the parameterization but also it will make it error free.

### Mockups or Images of the feature

![image](https://github.com/usebruno/bruno/assets/117565091/59542112-2992-41bb-a337-3e55444490a1)

**Comment by paolo-startselect** [comment]
I agree. This would really help when using env variables in the json request body and other places

**Comment by anusree-bruno** [comment]
This is available in latest versions

<img width="1477" height="885" alt="Image" src="https://github.com/user-attachments/assets/3803c180-8850-4c8e-81f7-4b2829edd155" />
